Source/ui_unix/openglwindow.cpp

```diff
@@ -0,0 +1,34 @@
+#include "openglwindow.h"
+#include <QResizeEvent>
+
+OpenGLWindow::OpenGLWindow(QWindow *parent)
+    : QWindow(parent)
+{
+    QSurfaceFormat format;
+    format.setVersion(3, 2);
+    format.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
Should use `CoreProfile` here. We want to be as close as possible to OpenGL ES and prevent usage of deprecated OpenGL functions.